CONTRIBUTING.md

```diff
@@ -88,11 +94,183 @@ We recommend the following commit structure in the following order:
 1. Refactoring (no functional change)
 1. Meaningful behavioural changes
 
-Follow [this guide](https://chris.beams.io/posts/git-commit/) to ensure you're writing good commit messages.
+Follow [Chris Beams' article on how to write a Git commit
+message](https://chris.beams.io/posts/git-commit/) to ensure you're writing
+good commit messages.
```

Comment:
It might be useful to add in examples of types for commit messages, as it is not mentioned here